plugin/plaintextmodel.cpp

```diff
@@ -24,16 +24,22 @@
 #include <QThread>
 #include <silicatheme.h>
 
+#include <unicode/ucsdet.h>
```

Comment:
Is this a necessary include for uchardet ?

Reply:
No, it isn't. I'll remove it